setup.py

```diff
@@ -36,17 +36,13 @@
 import vsc.install.shared_setup as shared_setup
 from vsc.install.shared_setup import ag, kh, jt, sdw, URL_GH_HPCUGENT
 
-# Re-reload the vsc modules that vsc-base ships
-shared_setup.RELOAD_VSC_MODS = True
```

Comment:
becasue it works without :smile: i probably made this workaround for something caused by another problem